src/applications/appeals/995/config/form.js

```diff
@@ -111,6 +112,8 @@ const formConfig = {
   defaultDefinitions: fullSchema.definitions,
   preSubmitInfo,
   submissionError,
+  // showReviewErrors: true,
```

Comment:
still need this comment?

Reply:
Yeah, eventually I'm going to uncomment it out and [show links in the error alert](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/docs/reviewErrors.md) on the review & submit page